README.md

```diff
@@ -12,25 +12,29 @@ This is the officially supported NodeJS library for using Adyen's APIs.
 
 ## Integration
```

Comment:
I find the title "Integration" a little confusing, I would prefer here a clear title like "Supported API versions".